runtime/rasdump/javadump.cpp

```diff
@@ -5263,6 +5263,7 @@ void
 JavaCoreDumpWriter::writeCPUinfo(void)
 {
 	PORT_ACCESS_FROM_PORT(_PortLibrary);
+	OMRPORT_ACCESS_FROM_J9PORT(_PortLibrary);
```

Comment:
Why do we need both of these? Which one is used for what?

Reply:
`j9sysinfo_get_number_CPUs_by_type` uses `PORT_ACCESS_FROM_PORT(_PortLibrary);`
`omrsysinfo_get_processor_feature_string` uses `OMRPORT_ACCESS_FROM_J9PORT(_PortLibrary);`

Reply:
Are we able to just use `omrsysinfo_get_number_CPUs_by_type`? Not sure what the J9 version provides here.

Reply:
yea i think we can. 
```cpp
#define j9sysinfo_get_number_CPUs_by_type(param1) OMRPORT_FROM_J9PORT(privatePortLibrary)->sysinfo_get_number_CPUs_by_type(OMRPORT_FROM_J9PORT(privatePortLibrary),param1)
```
- they are equivalent

Reply:
Everything in the OMR port library at the time OMR was created got j9 macros so the code didn't all need to be modified. No issues with using the omr variants directly.